webapp/actions/global_actions.jsx

```diff
@@ -596,3 +596,18 @@ export function redirectUserToDefaultTeam() {
         browserHistory.push('/select_team');
     }
 }
+
+export function updateOpenGraphMetaData(url) {
```

Comment:
Why is this function called `updateOpenGraphMetaData` when it's using `Client.getOpenGraphMetaData` also the word `MetaData` shouldn't be `Metadata` => `getOpenGraphMetadata`?